data/_models/shared/Variable mappings.csv

```diff
@@ -0,0 +1,14 @@
+tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles,visible
+Variable mappings,,,,,,,,,,,,"Mappings from collected variables to standard/harmonised variables, optionally including ETL syntax",DataCatalogueFlat,
+Variable mappings,,source,ref,1,TRUE,,Collections,,,,,collection that contains the source variable(s),DataCatalogueFlat,
+Variable mappings,,source dataset,ref,1,TRUE,,Datasets,source,,,,dataset that contains the source variable(s),DataCatalogueFlat,
+Variable mappings,,source variables,ref_array,,,,Variables,source dataset,,,,"optional, source variable that was mapped from. You may also indicate that a mapping to a target variable was not done and leave this field empty (match = not available)",DataCatalogueFlat,
```

Comment:
might we stop with reflink for target?

for source it might be that source doesn't want to document variables used.